### PR TITLE
Portfolio accessibility

### PR DESCRIPTION
**Pull request by Rustamxon7**
### We have created this pull request to check if there are any issues with accessibility!

1. [Page titles - one issue found](https://github.com/Rustamxon7/Microverse-portfolio-project/pull/21/files#r677415380)❎❗️
2. [Image text alternatives - one issue found](https://github.com/Rustamxon7/Microverse-portfolio-project/pull/21/files#r677416273) ❎❗️
3. Text headings - no issue found ✅
4. Color contrast - no issue found ✅
5. Resize - no issue found ✅
6. Interaction - no issue found ✅
7. [Moving content - one issue found](https://github.com/Rustamxon7/Microverse-portfolio-project/pull/21/files#r677416901) ❎❗️
8. Multimedia - no issue found ✅
9. The basic structure of the page - no issue found ✅

### CODE REVIEW

**Review by mikenath223 — CHANGES_REQUESTED** [review]
Hello Team,

You've done pretty well so far!
But for a few issues, you are almost done with this. Hang in there!


## Required Changes ♻️
_Please check the comments under the review._
- [ ] When an issue is found please link to the corresponding line of code in the Pull Request changes. So it's easy to trace the issue.

Happy coding, Cheers! 🚀🚀

Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 1]
```diff
@@ -4,7 +4,7 @@
     <meta charset="UTF-8" />
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>My Portfolio Project</title>
```
Comment (Rustamxon7) [inline index.html:7]: Page titles - one issue found

File: index.html [hunk 2]
```diff
@@ -32,7 +32,7 @@
   <body>
     <nav class="navigation">
       <p class="logo">My Logo</p>
-      <img src="images/Union.svg" alt="union" />
+      <img src="images/Union.svg" alt="logo" />
```
Comment (Rustamxon7) [inline index.html:20]: Image text alternatives - one issues were found

File: styles.css [hunk 3]
```diff
@@ -64,8 +64,8 @@ body {
 }
 
 .logo:hover {
-  transition-duration: 1s;
-  transform: rotate(360deg);
+  transform: scale(1.3);
+  transition-duration: 0.1s;
```
Comment (Rustamxon7) [inline styles.css:68]: Moving content - one issue found.

**Review by Rustamxon7 — COMMENTED** [review]
We have checked all issues in our project and fixed them.
Page title - one issue found.
Image text alternatives - one issue found.
Moving content - one issue found.

**Comment by Rustamxon7** [comment]
> Hello Team,
> 
> You've done pretty well so far!
> But for a few issues, you are almost done with this. Hang in there!
> 
> ## Required Changes ♻️
> _Please check the comments under the review._
> 
> * [ ]  When an issue is found please link to the corresponding line of code in the Pull Request changes. So it's easy to trace the issue.
> 
> Happy coding, Cheers! 🚀🚀
> 
> Do feel free to leave any questions or comments in the PR thread if something is not 100% clear.
> 
> **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, whether it is valid or invalid unless it is requested otherwise._**
> 
> _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

Thank you for reviewing my code... I did some changes and I guess linked these issues and reviewed but I guess I didn't understand you clear enough. Can you check my code again please!

File: index.html (hunk 1 above)
Comment (rloterh) [inline index.html:7]: - [ ] Good job for making all requested changes, however your **code hosted on GitHub pages has not been updated** to reflect the new changes. Kindly ensure to update your code on Github pages with your current code containing these changes.
- [ ] OPTIONAL
The demo link in your ReadMe file is appearing twice. Kindly ensure to remove the additional link.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Rostam and Ashley, 

You did great on this project, and for making the changes requested by the previous reviewer!👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there!👍 

## Good Points 👍  
- No linter errors

## Required Changes ♻️
- [x] _Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: styles.css (hunk 3 above)
Comment (rloterh) [reply styles.css:68]: Issue was well resolved! ✔️

File: index.html (hunk 2 above)
Comment (rloterh) [reply index.html:20]: Issue was well resolved! ✔️

File: index.html (hunk 1 above)
Comment (rloterh) [reply index.html:7]: Issue was well resolved! ✔️

Comment (Rustamxon7) [reply index.html:7]: I did all changes as you required... I hope there are not going to be any errors! It's my last attempt.

**Review by rloterh — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi Rostam and Ashley, 

Your project is complete! Great job for making the necessary changes recommended in the previous review(s). There is nothing else to say other than... it's time to merge it 🍾👍.
Congratulations! 🎉🥂


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**